_episodes/06-vertical_interpolation.md

```diff
@@ -3,9 +3,9 @@ title: "From hybrid sigma-pressure (model) levels to pressure levels"
 teaching: 0
 exercises: 0
 questions:
-- "How to interpolate on pressure levels?"
+- "How to interpolate pressure levels?"
```

Comment:
We don't really interpolate pressure levels. We interpolate other quantities such as temperature and zonal winds from hybrid sigma model levels to pressure levels.